vpype/io.py

```diff
@@ -429,10 +432,14 @@ def write_svg(
 
         for layer in corrected_doc.layers.values():
             for line in layer.pen_up_trajectories():
-                group.add(
-                    dwg.line((line[0].real, line[0].imag), (line[-1].real, line[-1].imag))
-                )
-
+                if no_basic_shapes:
+                    group.add(
```

Comment:
In my opinion, this feature should implement a single, monolithic `path` element, with multiple `M` commands.

Reply:
i try to figure out how i can do this